pkg/bot/interactive/message.go

```diff
@@ -28,15 +28,30 @@ const (
 type Message struct {
 	Type MessageType
 	Base
-	Sections          []Section
+
+	Sections []Section
+
```

Comment:
What is the logic behind having empty lines for some fields, bot not others?

Reply:
yep, as previously I had 
```go
// Message represents a generic message with interactive buttons.
type Message struct {
	Type MessageType
	Base

	Sections []Section
	Actions   []Actions

	OnlyVisibleForYou bool
	ReplaceOriginal   bool
}
```

But I simplified the struct, but we can remove those new lines, I don't have strong opinion here 👍